.github/workflows/acceptance_tests_reusable.yaml

```diff
@@ -87,7 +87,14 @@ jobs:
         with:
           ssh-key: ${{ secrets.bbl_ssh_key}}
       - name: Deploy autoscaler
+        shell: bash
         run: |
+          #! /usr/bin/env bash
+          set -eu -o pipefail
+
+          make --directory='./src/acceptance/assets/app/go_app' generate-fakes
+          make --directory='./src/autoscaler' generate-fakes
+
           cd "${AUTOSCALER_DIR}"
           make deploy-autoscaler
```

Comment:
Let's re-think the design here. Should we really depend for this make-target `deploy-autoscaler` on `go mod tidy`? Personally I think this is fine. But perhaps there are deviating opinions? @silvestre

Reply:
Yes, we should probably remove the dependency from `deploy-autoscaler` to `go-mod-tidy`, similarly as we also do not have a dependency from `deploy-autoscaler` to `package-specs`.

Reply:
Adaption done!